portal/common/lib/domain_parsing.test.ts

```diff
@@ -5,6 +5,8 @@ import { describe, expect, test } from 'vitest'
 import { getDomain, getSubdomainAndPath } from './domain_parsing'
 import { DomainDetails } from './types'
 
+const PORTAL_DOMAIN_NAME_LENGTH = 21
```

Comment:
question: what other alternatives do we have to avoid this rather ambiguous length? Specifying the full portal name? or the number of splits?

Reply:
I think the next right approach would be to specify the portal name, but this would also require to set on every vercel deployment the corresponding portal domain explicitly. 

Since all the subdomains of blocksite have the same length, this is an easy way of going around this.